### PR TITLE
Ma 4176/update region mapping

### PR DESCRIPTION
**Pull request by bhargavapratik**
MA-4176: Updated particle-region-mapping for aws-services.

### CODE REVIEW

**Review by mayankbharadwaj — APPROVED** [review]

**Comment by sandeshrajusungardas** [comment]
Approved, commenting as i don't have approve option.

File: particle-region-mapping [hunk 1]
```diff
@@ -1 +1,6 @@
-managed-os: ap-south-1  eu-west-3 eu-west-2 eu-west-1 ap-northeast-2 ap-northeast-1 sa-east-1 ca-central-1 ap-southeast-1 ap-southeast-2 eu-central-1 us-east-1 us-east-2 us-west-1 us-west-2 
+awsconfigrules: us-east-1 us-west-2
+awscfn-stack-launcher: us-east-1 us-west-2
+cloudtrail: us-east-1 us-west-2
+awsconfig: ap-south-1 eu-west-3 eu-west-2 eu-west-1 ap-northeast-2 ap-northeast-1 sa-east-1 ca-central-1 ap-southeast-1 ap-southeast-2 eu-central-1 us-east-1 us-east-2 us-west-1 us-west-2
```
Comment (RajkumarShivage) [inline particle-region-mapping:4]: Shouldn't it be awsconfigrules?

File: particle-region-mapping [hunk 2]
```diff
@@ -1 +1,6 @@
-managed-os: ap-south-1  eu-west-3 eu-west-2 eu-west-1 ap-northeast-2 ap-northeast-1 sa-east-1 ca-central-1 ap-southeast-1 ap-southeast-2 eu-central-1 us-east-1 us-east-2 us-west-1 us-west-2 
+awsconfigrules: us-east-1 us-west-2
```
Comment (RajkumarShivage) [inline particle-region-mapping:1]: Shouldn't it be awsconfig?

File: particle-region-mapping (hunk 1 above)
Comment (bhargavapratik) [reply particle-region-mapping:4]: yes, donce changes.

**Pull request merged**